main/acle.md

```diff
@@ -9382,6 +9415,2344 @@ are named after. All of the functions have external linkage.
+                                       svint8_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_s8]
+  //   _za32[_u8]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmlsl[_single]_za32[_s8]_vg4x4(uint32_t slice_base,
+                                       uint64_t slice_x4_offset, svint8x4_t zn,
+                                       svint8_t zm);
+  
+  ```
+
+#### SMLSLL, UMLSLL (multi)
+
+Multi-vector multiply-subtract long long (widening)
+
+``` c
+  // Variants are available for:
+  //   _za32[_s8]
+  //   _za32[_u8]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmlsl_za32[_s8]_vg4x2(uint32_t slice_base, uint64_t slice_x4_offset,
+                              svint8x2_t zn, svint8x2_t zm);
+  
+  
+  // Variants are available for:
+  //   _za32[_s8]
+  //   _za32[_u8]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmlsl_za32[_s8]_vg4x4(uint32_t slice_base, uint64_t slice_x4_offset,
+                              svint8x4_t zn, svint8x4_t zm);
+  
+  ```
+
+#### SMLSLL, UMLSLL (indexed)
+
+Multi-vector multiply-subtract long long (widening)
+
+``` c
+  // Variants are available for:
+  //   _za32[_s8]
+  //   _za32[_u8]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmlsl_lane_za32[_s8]_vg4x1(uint32_t slice_base, uint64_t slice_x4_offset,
+                                   svint8_t zn, svint8_t zm, uint64_t imm_idx);
+  
+  
+  // Variants are available for:
+  //   _za32[_s8]
+  //   _za32[_u8]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmlsl_lane_za32[_s8]_vg4x2(uint32_t slice_base, uint64_t slice_x4_offset,
+                                   svint8x2_t zn, svint8_t zm, uint64_t imm_idx);
+  
+  
+  // Variants are available for:
+  //   _za32[_s8]
+  //   _za32[_u8]
+  //   _za64[_s16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  //   _za64[_u16] (only if __ARM_FEATURE_SME_I16I64 != 0)
+  __attribute__((arm_streaming, arm_shared_za))
+  void svmlsl_lane_za32[_s8]_vg4x4(uint32_t slice_base, uint64_t slice_x4_offset,
+                                   svint8x4_t zn, svint8_t zm, uint64_t imm_idx);
+  
+  ```
+
+#### BFMLSLB, BFMLSLT
+
+BFloat16 floating-point multiply-subtract long from single-precision (top/bottom)
+
+``` c
+  
+  __attribute__((arm_streaming_compatible))
+  svfloat32_t svmlslb[_f32](svfloat32_t zda, svbfloat16_t zn, svbfloat16_t zm);
```

Comment:
These definitions clash with some of the existing fp16->fp32 mlslb/t intrinsics added for SVE2.

Could these be renamed so that they are different from the fp16 intrinsics, possibly to `svbfmlslb`?

Reply:
Thanks for pointing out! I've done this in https://github.com/ARM-software/acle/pull/217/commits/3b8a6e7ad94764182c28863aadc6a036803af2bd